docs/user/openstack/install_upi.md

```diff
@@ -526,20 +514,18 @@ openstack security group rule create --description "worker ingress services (UDP
 openstack security group rule create --description "VRRP" --protocol vrrp --remote-ip 192.0.2.0/24 "$INFRA_ID-worker"
 ```
 
-
 ### Router
 
 The outside connectivity will be provided by floating IP addresses. Your subnet needs to have a router set up for this to work.
 
 ```sh
-$ openstack router create "$INFRA_ID-external-router" --tag openshiftClusterID="$INFRA_ID"
-$ openstack router set --external-gateway <external> "$INFRA_ID-external-router"
-$ openstack router add subnet "$INFRA_ID-external-router" "$INFRA_ID-nodes"
+openstack router create "$INFRA_ID-external-router" --tag openshiftClusterID="$INFRA_ID"
+openstack router set --external-gateway <external> "$INFRA_ID-external-router"
+openstack router add subnet "$INFRA_ID-external-router" "$INFRA_ID-nodes"
```

Comment:
I'd prefer to have the shell prompts in because these command's can't be just copy/pasted.